WikiWatershed/model-my-watershed
# Fix DRB Point Source Listing

## Overview

For some edge cases, the query for whether it was "within" the DRB or not would fail. In order to ensure we capture all of them correctly, we now test against the simplified perimeter instead.

![image](https://user-images.githubusercontent.com/1430060/40320750-aee7579e-5cfa-11e8-9f2f-b39b21ba90ab.png)

Green is White Clay Creek HUC-10. Red is the DRB boundary. Blue is a simplified, buffered DRB boundary. 

In this PR, we switch from comparing the green against the red to comparing the green against the blue.

Connects #2812 

### Demo

![image](https://user-images.githubusercontent.com/1430060/40322812-8f35e95e-5d01-11e8-9bda-b5e409b8d0e5.png)

## Testing Instructions

* Check out this branch and restart Celery

      $ vagrant ssh worker -c 'sudo service celeryd restart'

* Analyze White Clay Creek HUC-10, or Upper Schuylkill HUC-10, or any other shape on the edge of the DRB. Go to the Point Sources tab. Turn on the related layer. Ensure the points on the map match those in the table.
* Analyze some shape outside of the DRB. Ensure it works as before.
